config.go

```diff
@@ -286,6 +286,11 @@ func NewSettings(filename string) (*Settings, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if err := settings.Validate(); err != nil {
```

Comment:
Actually, I'm not sure. @john-m-liu Is your build in staging failing as a result of this line?